src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const array = [...sourceString.split(';')];
+
+  array.map((element, i) => [
+    element.slice(0, element.indexOf(':') + 1),
+    element.slice(element.indexOf(':') + 1, -1),
+  ]);
+
+  array.map((element) => element[0] + element[1]);
```

Comment:
The second `map` function is attempting to concatenate the key and value strings, but this is not necessary for creating an object. Instead, you should focus on creating an array of key-value pairs.